DSL/DMapper/lib/helpers.js

```diff
@@ -0,0 +1,33 @@
+import { createHmac, timingSafeEqual } from "crypto";
+
+export function verifySignature(payload, headers) {
+  const signature = headers["x-hub-signature"];
+  const SHARED_SECRET = "wNc6HjKGu3RZXYNXqMTh";
```

Comment:
Is this a secret key @kalsaraM ? If so we have to add it to a secrets file and refer to it in the environments variable or through docker